File: src/main/kotlin/ru/leoltron/onmeeting/model/CardViewModel.kt
package ru.leoltron.onmeeting.model

import com.fasterxml.jackson.annotation.JsonInclude
import java.sql.Timestamp

@JsonInclude(JsonInclude.Include.NON_NULL)
data class CardViewModel(
        val title: String,
        val username: String,
        val cardId: Int,
        val locationString: String?,
        val startDate: Timestamp?,
        val endDate: Timestamp?,
        val participants: List<UserModel>,
        val tags: List<TagViewModel>)

File: src/main/front/src/models/CardViewModel.ts
import {UserModel} from "./userModel";
import {TagViewModel} from "./tagViewModel";

export interface CardViewModel {
    title: string,
    username: string,
    cardId: number,
    locationString: string,
    startDate: string,
    endDate: string,
    participants: UserModel[],
    tags: TagViewModel[]
}